Formula/d/dependabot-cli.rb

```diff
@@ -0,0 +1,31 @@
+class DependabotCli < Formula
+  desc "Tool for testing and debugging Dependabot update jobs"
+  homepage "https://github.com/dependabot/cli"
+  url "https://github.com/dependabot/cli/archive/refs/tags/v1.54.0.tar.gz"
+  sha256 "ae496c65e4f947cca1251c9f786fbcc1e5e1a079af7fee08b893265e69d39295"
+  license "MIT"
+
```

Comment:
```suggestion
```

Reply:
can you tell me where does this come from, have seen this couple of times recently

Reply:
I added it, since it seemed logically separate. Happy to remove though.

Reply:
we should always remove it, have seen that couple of times, that is why I asked.